.github/workflows/backport.yaml

```diff
@@ -0,0 +1,24 @@
+name: Automatic Backporting
+
+on:
+  pull_request_target:
+    types: ["labeled", "closed"]
+
+jobs:
+  backport:
+    name: Backport PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Backport Action
+        uses: sorenlouv/backport-github-action@v9.5.1
+        with:
+          github_token: ${{ secrets.ORG_REPO_TOKEN }}
+          auto_backport_label_prefix: backport-to-
```

Comment:
Is it possible to add options to return error for internal `sorenlouv/backport-github-action` errors?
e.g. we got error, but action finished successfully - https://github.com/DmitriyLewen/trivy/actions/runs/9364339038/job/25777130191#step:3:39

Reply:
I didn't find such an option. But as for this specific case, we can use `commitConflicts: true`.
https://github.com/sorenlouv/backport/blob/main/docs/config-file-options.md#commitconflicts

Reply:
Added https://github.com/aquasecurity/trivy/pull/6781/commits/72645dad3f30268d24f053ee513b322519fe1b72

Reply:
this doesn't work as we expected - https://github.com/DmitriyLewen/trivy/pull/82/files, https://github.com/DmitriyLewen/trivy/pull/85/files

Reply:
`commit Conflicts: true` includes these conflicts in code instead of returning error (https://github.com/DmitriyLewen/trivy/actions/runs/9379168342/job/25823651098)

Reply:
Yes, it's expected.

Reply:
You can see the document.

>When running backport on CI (aka in non-interactive mode) any commit conflicts will result in the backport being aborted. When commitConflicts: true even files with conflicts will be committed and a backport PR created. It is then up to a user to manually resolve the conflict, If auto-merge is enabled, this will be disabled for this PR specifically to avoid merging unresolved conflicts.

Reply:
IMO, it's better to open a PR anyway and resolve conflicts on the PR rather than aborting.

Reply:
It doesn't look very comfortable.
Can we move  table from  merge PR to the PR backport?
as here https://github.com/DmitriyLewen/trivy/pull/68#issuecomment-2146965089

Reply:
>Backport failed because of merge conflicts

Does it show this error even with `commitConflicts: true`?

Reply:
No. Backport just creates PR with  `<<<<< ` and `>>>>>` :
https://github.com/DmitriyLewen/trivy/actions/runs/9379290096/job/25823987233
https://github.com/DmitriyLewen/trivy/pull/85/files

Reply:
> resolve conflicts on the PR

This makes sense, but GH doesn't mark it as `conflict` (see https://github.com/DmitriyLewen/trivy/pull/85).
We can only see the problem in `tests`.
So, to fix this: you need to pull this branch => manually update these changes => push these changes.

Reply:
before this it was better because Backport told which PR you need to merge before that PR:
https://github.com/DmitriyLewen/trivy/pull/74#issuecomment-2147003892

Reply:
>This makes sense, but GH doesn't mark it as conflict (see https://github.com/DmitriyLewen/trivy/pull/85).

Yes, there is a risk of accidentally merging `<<<<<` and `>>>>>`, but I didn't find a good approach. Do you have any ideas? If `backport` aborts with conflicts (`commitConflicts: false`), it means we must manually create a backport PR with resolved conflicts.

- `commitConflicts: true`
    - pros:
        - No need to run `git cherry-pick` or `backport` locally
    - cons:
        - Risk of accidentally merging `>>>>>` as it is not treated as a conflict on GitHub
- `commitConflicts: false`
    - pros:
        - Low risk of accidentally merging `>>>>>` (although manual `git cherry-pick` may fail)
    - cons:
        - Need some manual work
            - `git cherry-pick` locally and resolve conflicts
            - Create a PR against `release/vX.Y`

Which is better?

Reply:
Let's say we added big changes for feature enhancement. We cannot backport the PR. If the critical fix is based on the change, the conflict cannot be avoided, and we have to resolve it manually.

Reply:
Okay, `commitConflicts: true` looks better to me.

perhaps we can create `comment` that this PR contains a conflict?

Reply:
>perhaps we can create comment that this PR contains a conflict?

It sounds like a good idea, but I didn't find the option.
https://github.com/sorenlouv/backport/blob/main/docs/config-file-options.md

Reply:
Perhaps we need to check changes in the commit...

Linter should show  error about this, so let's just leave `commitConflicts: true` and get some experience with it.
If necessary, we will think about checking the commit to reduce risks

Reply:
Hmm. It may be better to create our own backport tool.

Reply:
This takes time...

Let's try current version. `Test` workflow should protect us from mistakes

Reply:
I already started before seeing your comment 😄 
https://github.com/aquasecurity/trivy/pull/6781#issuecomment-2149450734